lithops/utils.py

```diff
@@ -373,6 +373,13 @@ def get_docker_path():
     return docker_path or podman_path
 
 
+def get_singularity_path():
+    singularity_path = shutil.which('singularity')
+    if not singularity_path:
+        raise Exception('singularity command not found. Install singularity')
+    return singularity_path
```

Comment:
This seems to be backend specific, so it would make more sense to put it in the config.py